admin/partials/boldgrid-backup-admin-premium.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * File: boldgrid-backup-admin-premium.php
+ *
+ * This file is used to markup premium cards page.
+ *
+ * @link https://www.boldgrid.com
+ * @since 1.12.4
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/admin/partials
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+defined( 'WPINC' ) || die;
+
+$nav = include BOLDGRID_BACKUP_PATH . '/admin/partials/boldgrid-backup-admin-nav.php';
+
+$premium = new \Boldgrid\Library\Library\Ui\Premium();
+
+$premium->enqueueScripts();
+
+$premium->cards = $this->get_cards();
+
+$premium_box = $this->core->config->is_premium_done ? '' : sprintf(
```

Comment:
This was also taken from other similar elements from other pages, such as ( boldgrid-backup\admin\partials\settings\storage.php ) . This is different than the standard wordpress dashboard notices, because it isn't something that is really dynamic or dismissable. This is always there if they're not premium.